hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -50,6 +50,10 @@
 import com.hedera.pbj.runtime.io.stream.ReadableStreamingData;
 import com.hedera.pbj.runtime.io.stream.WritableStreamingData;
 import com.hederahashgraph.api.proto.java.AccountID.AccountCase;
+import com.hederahashgraph.api.proto.java.FixedFee;
```

Comment:
I believe that we have, in this class, been trying to always import the hapi version, and fully qualify only the hederahashgraph version of protobuf classes.

Reply:
fixed it thanks